cpp/cmake_modules/UseCython.cmake

```diff
@@ -145,6 +145,8 @@ function(compile_pyx
             ${no_docstrings_arg}
             ${cython_debug_arg}
             ${CYTHON_FLAGS}
+            # Necessary for autodoc of function arguments
+            --directive embedsignature=True
```

Comment:
Nice! Can we remove the corresponding directives from the cython files?

Reply:
done, remove the directive from the files that had it